### DISCUSSION TITLE
Corner 8th as an option

### DISCUSSION THREAD

**Discussion by michaelochs** [rxhanson/Rectangle#879] [General]
There is an option to move a window to the left and right third or quarter of the display. There's also an option to move the window to e.g. the left third but then cut its height in half (top left sixth), however the same tiling doesn't exist for the quarter option. E.g. I want to move a window to the top left corner, quarter of the display's width and half the display's height.

This is super handy for ultra wide displays that are essentially two regular 27" screens next to each other in one display.

**Comment by rxhanson** [comment]
Thanks for the feedback. Check this: https://github.com/rxhanson/Rectangle/discussions/753
